expirationd.lua

```diff
@@ -322,6 +327,26 @@ local function expirationd_run_task(name, space_id, is_tuple_expired, options)
     end
     task.process_expired_tuple = options.process_expired_tuple or default_tuple_drop
 
+    -- validate index
+    local expire_index = box.space[space_id].index[0]
+    if options.index then
+        local index_exists = false
+        for _, index in pairs(box.space[space_id].index) do
```

Comment:
Why is `box.space[space_id].index[options.index]` not enough?
Probably I want to specify space_id, not space name - your approach doesn't allow it.

Reply:
> Probably I want to specify space_id, not space name

You probably meant options.index. Added this feature